examples/cs/grpc/scalar/svc.cs

```diff
@@ -0,0 +1,162 @@
+﻿
+//------------------------------------------------------------------------------
+// This code was generated by a tool.
+//
+//   Tool : Bond Compiler 0.11.0.0
+//   Input filename:  scalar.bond
+//   Output filename: scalar_grpc.cs
+//
+// Changes to this file may cause incorrect behavior and will be lost when
+// the code is regenerated.
+// <auto-generated />
+//------------------------------------------------------------------------------

```

Comment:
Why are we including this codegen file? The csproj will generate scalar_grpc.cs, can't seem to decipher the need for this.

Reply:
Ah. This is left-over from when I hand-wrote the initial streaming bindings to figure out what we needed to generate. I've reverted these changes in commit 6a29ad9b.